src/core/agent/agent.ts

```diff
@@ -264,20 +265,11 @@ class Agent {
   }
 
   private async getBran(): Promise<string> {
-    let bran: DataType | null;
-    try {
-      bran = await SecureStorage.get(KeyStoreKeys.SIGNIFY_BRAN);
-    } catch (error) {
-      if (
-        error instanceof Error &&
-        error.message ===
-          `${SecureStorage.KEY_NOT_FOUND} ${KeyStoreKeys.SIGNIFY_BRAN}`
-      ) {
-        bran = randomPasscode();
-        await SecureStorage.set(KeyStoreKeys.SIGNIFY_BRAN, bran);
-      } else {
-        throw error;
-      }
+    const bran = await SecureStorage.get(KeyStoreKeys.SIGNIFY_BRAN);
+    if (!bran) {
+      throw new Error(
+        `${SecureStorage.KEY_NOT_FOUND} ${KeyStoreKeys.SIGNIFY_BRAN}`
+      );
```

Comment:
Actually, this error will automatically be thrown if it's not found. So I think we can simplify and just have:

```
return (await SecureStorage.get(KeyStoreKeys.SIGNIFY_BRAN)) as string;
```